modules/oss/oss.c

```diff
@@ -187,7 +187,7 @@ static void *record_thread(void *arg)
 
 	while (st->run) {
 
-		n = read(st->fd, st->sampv, st->sampc*2);
+		n = (int)read(st->fd, st->sampv, st->sampc*2);
```

Comment:
I am not sure if we should maintain the oss module anymore, for modern linux alsa should be used.

is anyone using oss anymore ?

Reply:
I'm unsure if oss still a thing on unix systems (FreeBSD etc.).